CHANGELOG.md

```diff
@@ -19,4 +19,5 @@
 * [ENHANCEMENT] Add spanlogger package. #42
 * [ENHANCEMENT] Add runutil.CloseWithLogOnErr function. #58
 * [ENHANCEMENT] Optimise memberlist receive path when used as a backing store for rings with a large number of members. #76 #77
+* [ENHANCEMENT] Optimize `conflictingTokensExist` called by `NotifyMsg` to use a memory pool instead of allocating a map for each call #84
```

Comment:
I would merge this with the previous item given it's in the same release, user perspective is that "memberlist receive got better".

```suggestion
* [ENHANCEMENT] Optimise memberlist receive path when used as a backing store for rings with a large number of members. #76 #77 #84
```